app/components/application_record_card/style.scss

```diff
@@ -2,49 +2,85 @@
 
 .app-application-card {
   border-bottom: 1px solid govuk-colour("mid-grey");
-  margin-bottom: 15px;
-  padding-bottom: 15px;
+  margin-bottom: govuk-spacing(3);
+  padding-bottom: govuk-spacing(3);
 
   @include govuk-media-query($from: tablet) {
     display: flex;
     align-items: stretch;
   }
-}
 
-.app-application-card:first-of-type {
-  border-top: 1px solid govuk-colour("mid-grey");
-  padding-top: 15px;
-}
+  @include govuk-media-query($from: $mobile-small-end) {
+    display: flex;
+    justify-content: space-between;
+  }
 
-// Trainee column
-.app-application-card_col:nth-last-child(3) {
-  @include govuk-media-query($from: tablet) {
-    width: 40%;
-    padding-right: 15px;
+  .app-application-col-trainee-course {
+    @include govuk-media-query($from: $mobile-small-end) {
+      width: 60%;
+    }
+
+    @include govuk-media-query($from: $mobile-big-end, $until: desktop) {
+      width: 65%;
+      padding-right: govuk-spacing(3);
+      display: flex;
+      justify-content: space-between;
+    }
+
+    @include govuk-media-query($from: $govuk-page-width) {
+      width: 65%;
+      padding-right: govuk-spacing(3);
+      display: flex;
+      justify-content: space-between;
+    }
+
+    .app-application-col-trainee {
+      margin-bottom: govuk-spacing(2);
+      width: 100%;
+      padding-right: govuk-spacing(3);
+
+      @include govuk-media-query($from: $mobile-big-end, $until: desktop) {
+        width: 60%;
+      }
+
+      @include govuk-media-query($from: $govuk-page-width) {
+        width: 60%;
+      }
+    }
+
+    .app-application-col-course {
+      width: 100%;
+      padding-right: govuk-spacing(3);
+
+      @include govuk-media-query($from: $mobile-big-end, $until: desktop) {
+        width: 40%;
+      }
+
+      @include govuk-media-query($from: $govuk-page-width) {
+        width: 40%;
+      }
+    }
   }
-}
 
-// Course details column
-.app-application-card_col:nth-last-child(2) {
-  @include govuk-media-query($until: tablet) {
+  .app-application-col-status {
     margin-top: govuk-spacing(2);
-    margin-bottom: govuk-spacing(2);
-  }
-  @include govuk-media-query($from: tablet) {
-    width: 25%;
+
+    @include govuk-media-query($from: $mobile-small-end) {
+      width: 40%;
```

Comment:
I think this needs to be 35% to match the first column.

Reply:
This is for size 401px when it's two columns. The first column is 60% and the second is 40%, the same as the prototype.

Reply:
In the review app the columns are 65% and 40% respectively at desktop breakpoint - this presumably should add to 100%

Reply:
Updated to 35%